Makefile

```diff
@@ -40,73 +45,47 @@ install-clean:
+release:
+	@git tag -f -am "Release tag v$(VERSION)" v$(VERSION)
+	$(SKIP) @git push origin v$(VERSION)
 	@make release-helper
-	@make release-pachctl-custom
-	@echo "Release completed"
+	@make release-pachctl
+	@echo "Release $(VERSION) completed"
 
-custom-release: release-helper release-pachctl-custom
-	@echo 'For brew install, do:'
-	@echo "$$ brew install https://raw.githubusercontent.com/pachyderm/homebrew-tap/$(shell $(GOPATH)/bin/pachctl version --client-only)/pachctl@$(shell $(GOPATH)/bin/pachctl version --client-only | cut -f -2 -d\.).rb"
-	@echo 'For linux install, do:'
-	@echo "$$ curl -o /tmp/pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v$(shell $(GOPATH)/bin/pachctl version --client-only)/pachctl_$(shell $(GOPATH)/bin/pachctl version --client-only)_amd64.deb && sudo dpkg -i /tmp/pachctl.deb"
-	# Workaround for https://github.com/laher/goxc/issues/112
-	@git push origin :v$(shell $(GOPATH)/bin/pachctl version --client-only)
-	@git tag v$(shell $(GOPATH)/bin/pachctl version --client-only)
-	@git push origin v$(shell $(GOPATH)/bin/pachctl version --client-only)
-	@echo "Release completed"
-
-release-pachctl-custom:
-	@# Run pachctl release script w deploy branch name
-	@VERSION="$(shell $(GOPATH)/bin/pachctl version --client-only)" ./etc/build/release_pachctl.sh $(shell $(GOPATH)/bin/pachctl version --client-only)
+release-helper: docker-build docker-push docker-build-pipeline-build docker-push-pipeline-build
 
 release-pachctl:
 	@# Run pachctl release script w deploy branch name
-	@VERSION="$(shell $(GOPATH)/bin/pachctl version --client-only)" ./etc/build/release_pachctl.sh
-
-release-helper: release-version docker-build docker-push docker-build-pachctl docker-push-pachctl docker-push-pipeline-build
-
-release-version: install-clean
-	@./etc/build/repo_ready_for_release.sh
+	@goreleaser release -p 1 $(GORELSNAP) --release-notes=$(CHLOGFILE) --rm-dist -f goreleaser/pachctl.yml
 
 docker-build:
-	DOCKER_BUILDKIT=1 docker build \
-		--build-arg GO_VERSION=`cat etc/compile/GO_VERSION` \
-		--build-arg LD_FLAGS="$(LD_FLAGS)" \
-		$(DOCKER_BUILD_FLAGS) \
-		--progress plain -f Dockerfile.pachd -t pachyderm_build .
-
-	docker build $(DOCKER_BUILD_FLAGS) -t pachyderm/pachd etc/pachd
-	docker tag pachyderm/pachd pachyderm/pachd:local
-
-	docker build \
-		--build-arg GO_VERSION=`cat etc/compile/GO_VERSION` \
-		--build-arg LD_FLAGS="$(LD_FLAGS)" \
-		$(DOCKER_BUILD_FLAGS) \
-		-t pachyderm/worker etc/worker
-	docker tag pachyderm/worker pachyderm/worker:local
-
-docker-build-pachctl:
-	DOCKER_BUILDKIT=1 docker build \
-		--build-arg GO_VERSION=`cat etc/compile/GO_VERSION` \
-		--build-arg LD_FLAGS="$(LD_FLAGS)" \
-		--build-arg GC_FLAGS="$(GC_FLAGS)" \
-		$(DOCKER_BUILD_FLAGS) \
-		--progress plain -f Dockerfile.pachctl -t pachyderm/pachctl .
+	DOCKER_BUILDKIT=1 goreleaser release -p 1 $(GORELSNAP) --skip-publish --rm-dist -f goreleaser/docker.yml
```

Comment:
**Note for future**: This is probably outside the scope of this PR, but do you think it would make sense, in a subsequent PR, to set DOCKER_BUILDKIT inside of goreleaser if possible (so that `goreleaser release` doesn't necessarily need to be run from the Makefile)?